src/frontend/src/pages/WorkPackageForm/CreateWorkPackageCRForm.tsx

```diff
@@ -0,0 +1,27 @@
+import { WPFormType, validateWBS } from 'shared';
+import WorkPackageForm from './WorkPackageForm';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../utils/routes';
+import { useQuery } from '../../hooks/utils.hooks';
+import { WorkPackageApiInputs } from '../../apis/work-packages.api';
+
+const CreateWorkPackageCRForm: React.FC = () => {
+  const query = useQuery();
+  const wbsNum = query.get('wbs');
+  const history = useHistory();
+
+  if (!wbsNum) throw new Error('WBS number not included in request.');
+
+  const onSubmit = (paylaod: WorkPackageApiInputs) => {};
```

Comment:
payload*

Reply:
try console logging this payload in this method to test that it's correctly submitting the form